scalableminds/webknossos
# Indicate nml download

This PR adds visual progress feedback to nml downloads. The nml download is now done with `fetch` and then later saved manually. 
When triggering a download the download icon should change to a spinning loading icon until the download finished and a pop up to save the file shows up. 
While the download is processing, the cursor should indicate the loading process. Another triggered download, while one is already active, should be ignored.

### URL of deployed dev instance (used for testing):
- https://indicatenmldownload.webknossos.xyz

### Steps to test:
- Please check whether the described behavior applies to all possible downloads. The downloads can be found:
  -  (admin) project list
  -  (admin) task annotation
  -  (admin) task list
  -  (admin) task type
  -  (dashboard) task list
  -  (dashboard) explorativ annotations


### Issues:
- fixes #4131

------
